scope.md

```diff
@@ -19,3 +21,20 @@ hyper-specific methods for one type of data to general, do-it-all packages
 (e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the 
 two. If you're interested in submitting your data visualization package, please 
 open a pre-submission inquiry on first.
+
+## Python package technical scope
+
+pyOpenSci is still defining the technical scope criteria for package review.  
+Your package **may not be in technical scope** for us to review at this time if
+fits any of the out-of-technical-scope criteria listed below. 
+
+- Package is an out of sync fork of another package repository. 
+Sometimes we understand that a package maintainer may need to step down.  In that case, we strongly suggest that the original package owner, transfer the package to a new organization that would allow transfer of ownership of the package rather than 
```

Comment:
```suggestion
We understand that sometimes a package maintainer may need to step down from their role. In that case, we strongly suggest that the original package owner transfer the package to a new organization that would allow transfer of ownership of the package rather than 
```
- "Sometimes we understand" sounds like there are times we don't understand
- remove comma splice

Reply:
I think maybe we don't want to make a blanket statement about "out-of-sync forks" because my guess is it's highly likely people *will* be developing forks precisely *because* the original developers and/or maintainers have stopped updating or even responding to issues, pull requests, etc.

Maybe the criterion should be something like:  
"submissions should not be a fork of a package *that is itself still actively maintained*, that would require maintainers of the fork to have detailed knowledge of both the fork itself and the upstream package. Such a package structure severely limits the number of possible maintainers and contributors. In some cases, this could be a sign that code needs to be vendored, although vendoring code can cause its own complexities (see next bullet)"